core/src/test/java/org/elasticsearch/search/aggregations/metrics/InternalStatsTests.java

```diff
@@ -32,13 +32,12 @@
     @Override
     protected InternalStats createTestInstance(String name, List<PipelineAggregator> pipelineAggregators,
                                                Map<String, Object> metaData) {
-        long count = randomIntBetween(1, 50);
-        double[] minMax = new double[2];
-        minMax[0] = randomDouble();
-        minMax[0] = randomDouble();
-        double sum = randomDoubleBetween(0, 100, true);
-        return new InternalStats(name, count, sum, minMax[0], minMax[1], DocValueFormat.RAW,
-            pipelineAggregators, Collections.emptyMap());
+        long count = randomIntBetween(0, 50);
+        double min = randomDoubleBetween(-1000, 1000, false);
+        double max = randomDoubleBetween(-1000, 1000, false);
+        double sum = randomDoubleBetween(-1000, 1000, false);
```

Comment:
I think these ranges are also too small, we should probably test a large range of values since this aggregation will likely be used for large ranges. Mostly here I'm thinking this might catch bugs in the reduce logic rather than the serialisation logic.